src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -106,5 +106,64 @@ public void SarifTransformerTests_ToCurrent_MinimumWithTwoRuns()
             v2Log.Runs[1].Tool.Name.Should().Be("AssetScanner");
             v2Log.Runs[1].Tool.SemanticVersion.Should().Be("1.7.2");
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWithLogicalLocations()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""logicalLocations"": {
+                        ""collections::list::add"": {
+                          ""name"": ""add"",
+                          ""kind"": ""function"",
+                          ""parentKey"": ""collections::list""
+                        },
+                        ""collections::list"": {
+                          ""name"": ""list"",
+                          ""kind"": ""type"",
+                          ""parentKey"": ""collections""
+                        },
+                        ""collections"": {
+                          ""name"": ""collections"",
+                          ""kind"": ""namespace""
+                        }
+                      },
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            SarifLogVersionOne v1Log = GetSarifLogVersionOne(v1LogText);
+            var transformer = new SarifVersionOneToCurrentVisitor();
+            transformer.VisitSarifLogVersionOne(v1Log);
+
+            SarifLog v2Log = transformer.SarifLog;
+
+            v2Log.Runs.Should().NotBeNull();
+            v2Log.Runs.Count.Should().Be(1);
+
+            v2Log.Runs[0].LogicalLocations.Should().NotBeNull();
```

Comment:
Could be shorter with temp variable `IList<LogicalLocation> logicalLocations = v2Log.Runs[0].LogicalLocations;`

Reply:
Done